### ISSUE TITLE
Remove connection between categories in legend for the tutorials DAG

### ISSUE THREAD

**Issue by krishnakumarg1984** [dealii/dealii#9377]
In the landing page for the tutorials, in the legend for the tutorials DAG, a solid black line connects each category from left to right, i.e.

Basic techniques----advanced techniques---....... ----unfinished codes----code gallery

This creates a false impression (subtly appealing to the cognitive perception of the reader) that each category is to be perused in the specific order given by the legend, since people tend to associate information flow from left to right  (at least those who use left-to-right languages).  The fact that we have "unfinished codes" and "code gallery" at the far right tends to reinforce this implicit perception.

However, if we look at the tutorials' DAG, it is clearly not a prerequisite to finish all the orange (advanced techniques) boxes before starting a fluid mechanics tutorial for example. In fact, even in the video lectures, Prof Bangerth proceeds directly from Step-6 to Step-20.  Another example is Step-34.  The same goes for time-dependent problems, wherein step-23 can be studied after step-6.  

In summary, perhaps we should leave the task of conveying of the recommended flow to the DAG itself. The legend should merely be a list of colour-coded boxes and what they represent, I think.

**Comment by bangerth** [comment]
So just omit the black lines?

**Comment by krishnakumarg1984** [comment]
Yes. The solid interconnection lines between legend entries can be omitted without losing information, I think. 

Currently, one has to scroll a full screen and then some more to even see what these colours mean.

- Is there any way the legend could be stacked vertically next to the DAG itself? 
- If this is not possible, can we perhaps consider putting the legend before the DAG?

**Comment by bangerth** [comment]
#9382 removes the lines.

I played with integrating the legend into the graph, but it is too large to make sense:
![image](https://user-images.githubusercontent.com/7504421/72643884-e44fca00-392c-11ea-9745-f4d46436f254.png)

**Comment by bangerth** [comment]
Putting a box around it is also not very much better:
![image](https://user-images.githubusercontent.com/7504421/72644019-30027380-392d-11ea-88ee-6ee3d0324cb4.png)

**Comment by krishnakumarg1984** [comment]
Making a two-row legend box around the left should do it?

First row: 4 entries
second row: 3 entries

For a reference example of what I mean, please see [here](https://uk.mathworks.com/matlabcentral/mlc-downloads/downloads/e5702400-4a80-11e4-9553-005056977bd0/a7e7f935-5efd-4133-ada5-e2a91ec8878b/images/screenshot.png), [here](https://uk.mathworks.com/matlabcentral/mlc-downloads/downloads/submissions/37583/versions/1/screenshot.jpg), [here](https://uk.mathworks.com/matlabcentral/fileexchange/27389-simonhenin-columnlegend) and [here](https://uk.mathworks.com/matlabcentral/fileexchange/37583-legendgrid)

**Comment by bangerth** [comment]
The issue is that I have no control over where stuff is placed.

**Comment by krishnakumarg1984** [comment]
Ah. I think I understood the issue (maybe)

The node position attribute "pos" Only works with `neato` or `fdp` layout engines, not `dot`.
